### PR TITLE
Offset for Eye-Focus

### PR DESCRIPTION
**Pull request by Baromeus**
adds option to set the offset for the eye-focus (follow cursor) per character.

I added the option right below the others.

image

Also I changed the default values for PARAM_ANGLE_X and PARAM_ANGLE_Y  to the representative values.

**Information:**
I had some problems with Models that where already set - please try if this problem persists (I wasn't able to reproduce it)

### CODE REVIEW

File: index.js
Comment (Tony-sama) [inline index.js:1]: "live2d_model_param_mouth_form_01_select" id is not used, need to be removed.

File: live2d.js
Comment (Tony-sama) [inline live2d.js:1]: "||-0.45" probably need to be normalized to 45 since convertion is done int the min.js file.

File: ui.js
Comment (Tony-sama) [inline ui.js:1]: Same -0.45 convertion needed.

File: window.html
Comment (Tony-sama) [inline window.html:1]: "value="-45"" Default should be 45 in place of -45 I suppose :)

**Review by Tony-sama — CHANGES_REQUESTED** [review]
A few corrections needed.

File: index.js
Comment (Baromeus) [reply index.js:1]: fixed

File: live2d.js
Comment (Baromeus) [reply live2d.js:1]: fixed

**Comment by Baromeus** [comment]
Fixed the mentioned problems.

**Pull request merged**